__tests__/app.test.js

```diff
@@ -410,4 +409,30 @@ describe("GET /api/users", () => {
             expect(text).toEqual("{\"status\":500,\"error\":\"Internal Server Error\"}")
         })
     })
+})
+
+describe("GET /api/articles?topic=", () => {
+    test("Should return 200 and an array of articles with the same topic in the query", () => {
+        return request(app).get("/api/articles?topic=mitch").expect(200).then((response) => {
+            const articles = response.body.response.rows;
+            expect(Array.isArray(articles)).toBe(true);
+            articles.forEach((article) => {
+                expect(typeof article).toBe("object")
+                expect(article.topic).toBe("mitch");
+            })
+        })
+    });
+    test("Should return 400 and an error message if passed invalid topic query", () => {
+        return request(app).get("/api/articles?topic=nonsense").expect(400).then((response) => {
+            const text = response.error.text;
```

Comment:
Couldn't 'nonsense' be a topic one day?

Wouldn't one assume that anything could potentially be a topic and there for a test for a 400 is not needed and actually you should be testing for a 404 topic not found?